neuralmonkey/encoders/imagenet_encoder.py

```diff
@@ -97,56 +98,70 @@ def __init__(self,
 
```

Comment:
Že jako nejdřív zkontroluju, jestli to stojí za to a pak to začnu sypat do toho selfu.